Refactor & additions

dcrime opened the pull request:
Snooze ✔️ 
Remove a reminder ✔️ 
Remove all reminders ✔️ 
Multiple time indicators ✔️ (`$remind add 5h 3m 2s`)

Hope I caught all the bugs and the code is good enough 😄 

Bugs:
Anything above a month is going to be offset due to me not understanding how time works

dcrime committed:
Refactor, ability to remove reminders, snooze

Refactored the code in hopes of making it easier to read

Added a faulty system to tell you how long until your reminder

Added the ability to remove a reminder or to remove all reminders

Added a snooze button which snoozes for 5 min

Hopefully I caught most of my bugs

dcrime committed:
Update README.md

Killerjet101 commented:
Sorry for the late response, I'll go ahead and look at it later today and maybe even fix the bug you got. Thanks so much!

fluzzeon committed:
Much needed update

-Fix months & years (I think)
-Updated license and readme
-Updated version number
